setup.py

```diff
@@ -67,5 +67,9 @@
 
     install_requires=requirements,
 
+    extras_require={
+        'Markdown_HTML': ['mistune>=0.8.4'],  # For status view markdown rendering
```

Comment:
Similar to my comment above, but also, status rendering is such an essential feature that any related requirement should not be specified as an `extra` IMO.

Reply:
Agreed, I also noted that the mistune text renderer you wrote now requires bs4 and tabulate, is it the plan to add those requirements as well?

Reply:
Yes, but I was planning to reevaluate this after we decide to move forward with this.